src/scripts.js

```diff
@@ -46,21 +46,23 @@ var userStairsThisWeek = document.getElementById('userStairsThisWeek');
 var userMinutesThisWeek = document.getElementById('userMinutesThisWeek');
 var bestUserSteps = document.getElementById('bestUserSteps');
 var streakList = document.getElementById('streakList');
-var streakListMinutes = document.getElementById('streakListMinutes')
+var streakListMinutes = document.getElementById('streakListMinutes');
+
+window.onload = startApp(); //Starts Here (global variables currentInformation = {})
 
 function startApp() {
-  let userList = [];
+  let userList = []; // let userRepo = new UserRepo(makeUsers()); next three lines ** here to randomHistory should be a function for instatiating data
   makeUsers(userList);
   let userRepo = new UserRepo(userList);
-  let hydrationRepo = new Hydration(hydrationData);
-  let sleepRepo = new Sleep(sleepData);
-  let activityRepo = new Activity(activityData);
-  var userNowId = pickUser();
-  let userNow = getUserById(userNowId, userRepo);
-  let today = makeToday(userRepo, userNowId, hydrationData);
-  let randomHistory = makeRandomDate(userRepo, userNowId, hydrationData);
-  historicalWeek.forEach(instance => instance.insertAdjacentHTML('afterBegin', `Week of ${randomHistory}`));
-  addInfoToSidebar(userNow, userRepo);
+  let hydrationRepo = new Hydration(hydrationData); // created our hydration class based on (need to get hydration from API)
+  let sleepRepo = new Sleep(sleepData); // same as above Sleep
+  let activityRepo = new Activity(activityData); // same as above Activity
```

Comment:
Are we making these global?

Reply:
I think we are going to try and keep them as it (with some adjustments). Making them global is definitely an option if you feel like that would make more sense to you.